app/[locale]/faculty-and-staff/[id]/page.tsx

```diff
@@ -1,9 +1,288 @@
-import WorkInProgress from '~/components/work-in-progress';
-
-export default function FacultyOrStaff({
-  params: { locale },
-}: {
-  params: { locale: string; id: string };
-}) {
-  return <WorkInProgress locale={locale} />;
+'use client';
```

Comment:
Don't make this entire file client-side. Abstract out the currentTab functionality to another file as a component and then import that.